cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -442,29 +442,12 @@ func gcloudProject() ([]string, error) {
 	return []string{cfg.Configuration.Properties.Core.Project}, nil
 }
 
-// Main executes the main function of the proxy, allowing it to be called from tests.
-//
-// Setting timeout to a value greater than 0 causes the process to panic after
-// that amount of time. This is to sidestep an issue where sending a Signal to
-// the process (via the SSH library) doesn't seem to have an effect, and
-// closing the SSH session causes the process to get leaked. This timeout will
-// at least cause the proxy to exit eventually.
-func Main(timeout time.Duration) {
-	if timeout > 0 {
-		go func() {
-			time.Sleep(timeout)
-			panic("timeout exceeded")
-		}()
-	}
-	main()
-}
-
-func main() {
+func startProxy() int {
```

Comment:
nit: This is probably more like `runProxy` since it doesn't return until complete

Reply:
Done.